cli.js

```diff
@@ -34,7 +43,7 @@ if (!input && process.stdin.isTTY) {
 }
 
 if (input) {
-	init(readChunk.sync(cli.input[0], 0, fileType.minimumBytes));
+	initFromFilePath(input);
 } else {
-	process.stdin.once('data', init);
+	process.stdin.once('data', initFromBuffer);
```

Comment:
Use the stream method on `file-type` and pipe `process.stdin`. We can no longer assume one `data` event is enough.